src/prefect/utilities/tasks.py

```diff
@@ -182,6 +184,17 @@ def preprocess(a: Any) -> "prefect.Task":
     return res
 
 
+# DEPRECATED backward-compatible import
```

Comment:
@jlowin Could you add this deprecation about the new import path to the changes even though it's small